src/workloads/execution/LastPoint.yml

```diff
@@ -0,0 +1,188 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests the performance of last point queries after the first milestone.
+
+Actors:
+- Name: CreateTimeSeriesCollection
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Operation:
+      OperationMetricsName: CreateTimeSeriesCollection
+      OperationName: RunCommand
+      OperationCommand:
+        {create: &coll Collection6, timeseries: {timeField: "time", metaField: "tags"}}
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *db
+  Phases:
+  - Nop: true
+  - Repeat: 1
+  - Nop: true
+  - Repeat: 1
+  - Nop: true
+  - Repeat: 1
+  - Nop: true
+  - Repeat: 1
+  - Nop: true
+  - Repeat: 1
+  - Nop: true
+  - Repeat: 1
+  - Nop: true
+
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Nop: true
+  - Nop: true
+  - Repeat: 1
+    Database: *db
+    Collection: *coll
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 10000
+    BatchSize: &batchSize 100
+    Document:
+      part1: {^Choose: {from: [1, 2, 3, 4]}}
+      part2: {^Choose: {from: [1, 2, 3, 4]}}
+      time: {^RandomDate: {
+        min: {^Inc: {start: 0, multiplier: 1, step: 1}},
+        max: {^Inc: {start: 1, multiplier: 1, step: 1}}
+      }
+      }
+      double: {^Choose: {from: [
+        {^RandomDouble: { min: 0.0, max: 500.0 }},
+        null], weights: [2, 8]}}
+      tags: { value: {^Choose: {from: [
```

Comment:
That's a good idea! I was thinking I could decrease the range in order to have values grouped closer to each other, did you have anything else in mind?

Reply:
Ah I see that this section is mostly taken from the $fill workload. In general I would make the model generation mirror a typical user workload, so in the $fill case it makes sense that there are so many null values. Your initial thinking is heading in the right direction - in this case, since we want to model a time-series usecase, I would look towards the [tsbs repo](https://github.com/gregorynoma/tsbs) for inspiration - with the caveat that we don't need as many fields in our workload as they have.

An example generated tsbs document:
```
{
	"time" : ISODate("2016-01-01T00:00:20Z"),
	"tags" : {
		"arch" : "x86",
		"datacenter" : "eu-central-1a",
		"hostname" : "host_0",
		"os" : "Ubuntu15.10",
		"rack" : "6",
		"region" : "eu-central-1",
		"service" : "19",
		"service_environment" : "test",
		"service_version" : "1",
		"team" : "SF"
	},
	"usage_irq" : 64,
	"measurement" : "cpu",
	"_id" : ObjectId("60dcd137a6ccbe8dc147aa34"),
	"usage_user" : 57,
	"usage_nice" : 61,
	"usage_softirq" : 7,
	"usage_steal" : 45,
	"usage_guest" : 81,
	"usage_guest_nice" : 36,
	"usage_system" : 0,
	"usage_iowait" : 22,
	"usage_idle" : 25
}
```

Reply:
Gotcha, I've modified the generation mirror to look more like this! I haven't included all the fields since we aren't really using them but would be happy to add anything you feel is needed!